fpga.chart.py

```diff
@@ -177,12 +191,20 @@ def _get_fpga_power(self, idx):
             return self._get_xilinx_fpga_power(idx)
 
 
+    def get_fpga_os_status(self):
+        while True:
+            time.sleep(self.temp_power_update_interval)
```

Comment:
This does not lead to a proper interval. You'll have to measure the time it took and subtract that.

Reply:
It's different for different cards, the xilinx card is very quick, the intel takes 1.5 seconds